tests/phpunit/api/v3/SyntaxConformanceTest.php

```diff
@@ -1057,6 +1057,12 @@ public function testCreateSingleValueAlter($entityName) {
             $optionValue = $this->callAPISuccess('option_value', 'create', array('option_group_id' => $optionGroupID, 'label' => 'new option value'));
             $options['values'][] = $optionValue['id'];
           }
+          if (isset($specs['pseudoconstant']['table'])) {
+            //some psedoconstant property is based on other entity attributes
```

Comment:
I'm not sure about this - perhaps @totten might have an opinion - but I think we are already testing some non-option-value pseudoconstants here & this reduces our test coverage of those. In theory there is no reason for non-option-value pseudoconstatnts not to work as the createTestObject should create them - although I'm not going to argue that it necessarily does what it should do.